src/aap_eda/services/activation/engine/kubernetes.py

```diff
@@ -167,7 +167,7 @@ def get_status(self, container_id: str) -> ContainerStatus:
                     status=ActivationStatus.FAILED,
                     message=container_status.state.terminated.message or "",
                 )
-                LOGGER.info(
+                LOGGER.warning(
                     f"Pod exited with {exit_code}, reason "
```

Comment:
Should also this message be: "Pod exited with $code. Reason: ..." like the similar messages in activation_manager.py?

Reply:
updated